sunpy/visualization/draw.py

```diff
@@ -0,0 +1,106 @@
+"""
+This module provides functions that draw on Map plots.
+"""
+import numpy as np
+from matplotlib.lines import Line2D
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.coordinates import HeliographicStonyhurst
+from sunpy.visualization import wcsaxes_compat
+
+__all__ = ["draw_equator", "draw_prime_meridian"]
+
+
+def draw_equator(axes, resolution=300, **kwargs):
+    """
+    Draws the equator as seen by the axes observer.
+
+    Parameters
+    ----------
+    axes : `matplotlib.axes.Axes`
+        The axes to plot the equator on.
+    resolution : `int`
+        The number of points used to represent the equator.
+
+    Examples
+    --------
+    >>> import sunpy.map
+    >>> from sunpy.visualization import draw
+    >>> import sunpy.data.sample   # doctest: +REMOTE_DATA
+    >>> aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)   # doctest: +REMOTE_DATA
+    >>> aia.plot()   # doctest: +SKIP
+    >>> draw.draw_equator(aia._check_axes(None))   # doctest: +SKIP
```

Comment:
This is how someone has to get the axes from the plot method?

Reply:
Well, they can, but they shouldn't (because it is a private method).  If we have some reason to avoid showing the explicit creation of axes in the docstring example, the best option is to call `plt.gca()` to retrieve the current axes.